_search-plugins/search-pipelines/neural-sparse-query-two-phase-processor.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: default
+title: NeuralSparse query two-phase processor
+nav_order: 13
+has_children: false
+parent: Search processors
+grand_parent: Search pipelines
+---
+
+# NeuralSparse query two-phase processor
+
+The `neural_sparse_two_phase_processor` search request processor is designed to set a speed-up pipeline for [neural sparse search]({{site.url}}{{site.baseurl}}/search-plugins/neural-sparse-search/). It accelerates the neural sparse query by breaking down the original method of scoring all documents with all tokens into two steps. In the first step, it uses high-weight tokens to score the documents and filters out the top documents; in the second step, it uses low-weight tokens to fine-tune the scores of the top documents.
```

Comment:
fine-tune is usually used for ML models, maybe we should use rescore here

Reply:
Replaced with word rescore.